tests/test_models.py

```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase
+
+from taxi.models import Driver
+
+
+class ModelTests(TestCase):
+    def setUp(self) -> None:
+        Driver.objects.create(
+            license_number="ACB1234",
```

Comment:
The `license_number` provided here is only 7 characters long. According to the validation logic, it should be 8 characters long. Consider updating it to meet the validation requirements.